src/map/pc.c

```diff
@@ -9456,6 +9456,18 @@ bool pc_equipitem(struct map_session_data *sd,short n,int req_pos)
 		return false;
 	}
 
+	// dont equip arrow if no bow is equipped
+	if ((sd->class_&MAPID_BASEMASK) == MAPID_ARCHER || (sd->class_&MAPID_BASEMASK) == MAPID_ARCHER_HIGH || (sd->class_&MAPID_BASEMASK) == MAPID_SNIPER || (sd->class_&MAPID_BASEMASK) == MAPID_HUNTER || (sd->class_&MAPID_BASEMASK) == MAPID_RANGER){
```

Comment:
This only applies to the Hunter class? What about the Stalker or Dancer/Bard classes?